src/bind.js

```diff
@@ -18,8 +18,14 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...args) {
+  function bindArgs(...additionalArgs) {
```

Comment:
good, but you can write simply
```suggestion
  return (...numbers) => callback(...arg, ...numbers);
```